lib/rules/function-url-quotes/__tests__/index.js

```diff
@@ -185,6 +185,9 @@ testRule(rule, {
     },
     {
       code: 'a { background: url("") }'
+    },
+    {
```

Comment:
Can we please pull this out into its own `testRule` block with `syntax: styled`, please?

We should place it at the end of the file.